rules/unauthenticated-service/raw.rego

```diff
@@ -29,7 +29,9 @@ deny contains msga if {
 		"packagename": "armo_builtins",
 		"alertObject": {"k8sApiObjects": [wl]},
 		"relatedObjects": [
-			{"object": service},
```

Comment:
I think we should also remove the "path" from line 27 as well..

Reply:
done

Reply:
The list should be empty (because we don't offer a fix to the main wl)
`"reviewPaths": []
`